lms/static/js/spec/student_profile/badge_list_container_spec.js

```diff
@@ -0,0 +1,85 @@
+define(['backbone', 'jquery', 'underscore', 'URI', 'common/js/spec_helpers/ajax_helpers',
+        'js/spec/student_profile/helpers',
+        'js/student_profile/views/badge_list_container',
+        'common/js/components/collections/paging_collection'
+    ],
+    function (Backbone, $, _, URI, AjaxHelpers, LearnerProfileHelpers, BadgeListContainer, PagingCollection) {
+        "use strict";
```

Comment:
Nit: this file mixes single quotes and double quotes. Our best practice is to use single quotes except for strings that themselves contain single quotes.